pytest_fixtures/api_fixtures.py

```diff
@@ -31,12 +44,7 @@ def module_location(module_org):
 
 
 @pytest.fixture(scope='module')
-def module_lce(module_org):
-    return entities.LifecycleEnvironment(organization=module_org).create()
-
-
-@pytest.fixture(scope='module')
-def module_smart_proxy(module_location):
+def default_smart_proxy(module_location):
```

Comment:
Some docblocks on these would be incredibly helpful, especially considering this and `default_domain` are actually returning data structures (the result of `read()`), not entity instances.

Reply:
No, `Read` returns instance itself of an entity. Same with `Create` which inside returns `Read` only. 

All `default_` fixtures are reading the existing entities from satellite instead of creating them, but they are returning an instance itself.